src/libstd/sys/sgx/waitqueue.rs

```diff
@@ -10,7 +9,10 @@ use crate::num::NonZeroUsize;
 /// Since userspace may send spurious wake-ups, the wakeup event state is
 /// recorded in the enclave. The wakeup event state is protected by a spinlock.
 /// The queue and associated wait state are stored in a `WaitVariable`.
```

Comment:
This should be an internal doc comment (with `//!` instead of `///`). That's why the `use` got moved up accidentally by some formatting tool.